schemas/dab.draft.schema.json

```diff
@@ -277,8 +277,33 @@
               "additionalProperties": false,
               "properties": {
                 "provider": {
-                  "type": "string",
                   "description": "The name of authentication provider",
+                  "oneOf": [
+                    {
+                      "const": "StaticWebApps",
+                      "description": "Authentication provided by Azure Static Web Apps."
+                    },
+                    {
+                      "const": "EntraID",
+                      "description": "Authentication provided by Microsoft Entra ID (formerly Azure AD)."
```

Comment:
Perhaps, mention in the description of EntraID as well that `jwt` property is required to configure similar to `Custom`?

Reply:
Good feedback, done!